update_versions.yml

```diff
@@ -9,13 +9,13 @@
 - name: get archive versions
   hosts: frontend
   tasks:
-    - shell: /var/cnx/venvs/archive/bin/pip freeze | grep 'cnx-archive'
+    - shell: /var/cnx/venvs/python2/archive/bin/pip freeze | grep 'cnx-archive'
```

Comment:
During the transition period, is it possible for this to report the wrong version if (for whatever reason) we have to revert some environment to use `/var/cnx/venvs/archive/bin/pserve` and (again, for whatever reason) the version in `environments/__prod_envs/vars/versions.yml` doesn't match `environments/__prod_envs/files/archive-requirements.txt`? 

If so, is it possible to somehow base the pip on `{{ archive_pserve_path }}`?

Reply:
I'm not removing the old virtualenv so I don't think the scenario you are describing would happen. If a revert did occur  it would go back to the old virtualenv, no? I could be missing something.

Maybe we should have a quick call?